packages/firestore/package.json-e

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Was this committed by mistake? I don't understand its significance.

Reply:
These are mistakes. I have no idea where they came from. I have removed them

Reply:
Mac…happened to me when I was using grep or find or something.